CHANGELOG.md

```diff
@@ -8,12 +8,16 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ### Fixed
 
+- JointModel -> Model::JointModel in algorithm/model.hxx for compilation disambiguation.
+- Return Vector3 instead of Vector3::ConstandReturnType in TransformHelicalTpl.
+- Do not specify size of Identity for constant size matrices.
```

Comment:
This changelog entries are not really helpful for users.

Instead of copy/pasting commit message, describe what your PR is fixing so end user will know if the new pinocchio version will solve their issues. And don't forget the link to the PR so user can have more details.

Example:
- Fix build issue with GCC XXX
- Fix appendModel build when called with XXX